.github/workflows/lspE2E.yml

```diff
@@ -73,8 +73,9 @@ jobs:
     with:
       automationBranch: ${{ inputs.automationBranch }}
       testToRun: 'visualforceLsp.e2e.ts'
-      vscodeVersion: ${{ inputs.vscodeVersion || '1.85.2' }}
+      vscodeVersion: ${{ inputs.vscodeVersion }}
       runId: ${{ inputs.runId }}
+      os: '["macos-latest", "ubuntu-latest"]'
```

Comment:
We're still running CJS for LSP category.  It takes some reruns, but Visualforce does pass on Windows in CJS.

Visualforce can be separately tested on Mac and Ubuntu in the ESM Test Suite for 1.86.